Source/Modules/php.cxx

```diff
@@ -133,6 +137,18 @@ extern "C" {
   static void (*r_prevtracefunc) (const SwigType *t, String *mangled, String *clientdata) = 0;
 }
 
+/* Function used to determine if its a valid object pointer that needs object creation.
+ * Used to mainly distingush between standard datatype pointers and typedef pointers.
+ */
+static bool SWIG_is_valid_class(String *Type_class) {
+  Iterator c_iterator;
+  for (c_iterator = First(classes); c_iterator.item; c_iterator = Next(c_iterator)) {
+    if (Cmp(c_iterator.item,Type_class) == 0)
+      return true;
+  }
+  return false;
+}
```

Comment:
I'm not really following the code where this is used, but I'm pretty certain this isn't the right approach to any problem you're needing to solve.  SWIG should be able to tell you if a type is a class or not - probably you want `SwigType_isclass(const SwigType *t)`.